Dockerfile

```diff
@@ -18,18 +18,18 @@ WORKDIR /home/gradle/diva
 RUN gradle ziptask
 RUN jar xvf build/distributions/diva-all.zip
 
-FROM openjdk:8u282-jre-slim
+FROM eclipse-temurin:17-jre-alpine
```

Comment:
What is your reason for needing java17?  We are tentatively using java8 as official runtime for two reasons. 

1. WALA analysis depends on builtin class files inside JRE, which differs between versions.  I think WALA is officially only tested with java11, although we removed the class version constraint, e.g., by the following code snippet, so that Diva accepts recent version of class files. 
https://github.com/konveyor/tackle-diva/blob/c7bd5f23f93554aded8d45f5dc0782252395c82c/core/src/main/java/io/tackle/diva/irgen/DivaIRGen.java#L630-L638
2. The forge/furnace library, required to run windup-integrated version of Diva, e.g., Dockerfile.windup,  seems like working only with java8.

Reply:
@akihikot You are welcome!

Tackle-DiVA crashed while analyzing more recent (Java 9+) apps. Java 8 has been released in 2014.

For this reason, I thought that I would give a try to a Java upgrade and I was very positively surprised to see it working properly without further change :-)

Reply:
@Maarc 
> Tackle-DiVA crashed while analyzing more recent (Java 9+) apps.
It would be helpful if you could provide us some info about your crash.   

Regarding java version, what about using java11 as it is officially supported version by wala-1.5.7?

Reply:
Sorry @akihikot, I missed your message.

Using Java 11 sounds just right. I don't have the error messages anymore. I just tested the latest version and it looks fine!